lib/resty/roundrobin.lua

```diff
@@ -9,16 +9,41 @@ local math_random = math.random
 local _M = {}
 local mt = { __index = _M }
 
+local new_tab
```

Comment:
Better to avoid repeating the code. We can put this stuff into a common module.

Reply:
already move this code to utils module